arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -114,7 +114,7 @@
           {
             "name": "$filter",
             "in": "query",
-            "required": false,
+            "required": true,
```

Comment:
Yes, this is describing that version of the api.  We do have a new version out,  but I wanted to correct the existing spec before I created the new one.

There is still some churn around the UsageAggregates API that needs to be worked out prior to leaving preview.